packages/trackerless-network/src/logic/StreamEntryPointDiscovery.ts

```diff
@@ -196,8 +197,8 @@ export class StreamEntryPointDiscovery {
             if (this.config.streams.has(streamPartId)) {
                 const stream = this.config.streams.get(streamPartId)
                 const rediscoveredEntrypoints = await this.discoverEntryPoints(streamPartId)
-                await stream!.layer1!.joinDht(rediscoveredEntrypoints, false)
-                if (stream!.layer1!.getBucketSize() < 4) {
+                await stream!.layer1!.joinDht(rediscoveredEntrypoints, false, false)
+                if (stream!.layer1!.getBucketSize() < this.config.networkSplitAvoidanceLimit) {
                     throw new Error(`Network split is still possible`)
                 }
```

Comment:
Should there be a check here of whether `stream.layer1 !== undefined`

Reply:
As the stream object is held in memory before await call the worst case here is that the layer1-node is stopped.